deposit-transactions.md

```diff
@@ -0,0 +1,165 @@
+# Deposit Transactions
+
+Although the L2 execution engine is nearly identical to that of L1, an important difference is the
+introduction of a new 'Deposit' Transaction Type. Deposit transactions have the following notable
+distinctions from existing transaction types:
+
+1. They are initiated by the System as part of the protocol.
+2. Do not require signature validation.
+
+## Deposit Transaction Type
+
+We define a new [EIP-2718] compatible transaction type with the prefix `0x7E`, and the following
+fields:
+
+[EIP-2718]: <https://eips.ethereum.org/EIPS/eip-2718>
+
+<!-- ToDo: set to more GoLang like type defs? -->
+- `address to`
+- `address from`
+- `uint256 value`
+- `bytes data`
+
+This transaction type contains a subset of the fields used in [EIP-155], but does not include
+signature information.
+
+[EIP-155]: https://eips.ethereum.org/EIPS/eip-155
+
+Although in practice we define only one new Transaction Type we can distinguish between two distinct
+transactions which occur in the deposit block, based on their positioning. The first transaction
+MUST be the [L1 Attributes Deposit Transaction][l1-attributes-deposit-transaction], followed by a
+dynamic array of [Deposited Transactions][deposited-transactions] submitted to the Deposit Feed
+contract by accounts on L1.
```

Comment:
- I would name the new transaction type, so we can refer to it by name here (and later?)
- There is no notion of "deposit block" at this stage of the spec.
- Are we calling them "deposited transactions" or "deposits"?
- Also shows that this term should have been defined earlier since it's being used here :)
- I'd say "zero or more" instead of a "a dynamic array of" which seems to imply things on storage format.